python/mxnet/symbol/symbol.py

```diff
@@ -1285,6 +1285,7 @@ def _get_ndarray_inputs(arg_key, args, arg_names, allow_missing):
             raise TypeError('Only accept list of NDArrays or dict of str to NDArray')
         return c_array(NDArrayHandle, arg_handles), arg_arrays
 
+    # pylint: disable=too-many-locals
```

Comment:
Obsolete due to global disable

Reply:
It defies explanation but these are actually required or the tests fail.